modules/internal/DefaultRectangular.chpl

```diff
@@ -1214,15 +1214,16 @@ module DefaultRectangular {
                              param getShifted = true) {
       if stridable {
         var sum = 0:intIdxType;
-        for param i in 1..rank do
+        for param i in 0..rank-1 do
           sum += (chpl__idxToInt(ind(i)) - chpl__idxToInt(off(i))) * blk(i) / abs(str(i)):intIdxType;
+        //        writeln("C: sum", sum);
```

Comment:
remove

Reply:
https://github.com/chapel-lang/chapel/pull/14522/commits/d6870e28c70082966a83a5c275f92da999176305 addressed this and the next few similar cases.